context/app/static/js/components/detailPage/provenance/ProvSection/ProvSection.jsx

```diff
@@ -16,14 +19,24 @@ function ProvSection({ iconTooltipText }) {
   if (isLoading) {
     return (
       <DetailPageSection id="provenance">
-        <SectionHeader>Provenance</SectionHeader>
+        <StyledSectionHeader>
+          Provenance
+          <SecondaryBackgroundTooltip title="The provenance shows the sequence of events and actions that led to this page creation.">
```

Comment:
```suggestion
          <SecondaryBackgroundTooltip title="The provenance shows the sequence of events and actions that led to this page's creation.">
```
@tsliaw ?

Reply:
Yes please add the apostrophe. Will fix in the Figma.